src/core/buffers/events_generators.ts

```diff
@@ -159,7 +174,7 @@ const EVENTS = {
 
   periodBufferCleared(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Update or refactor this function so that its implementation doesn't duplicate the one on line 147. ([typescript:S4144](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4144))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=377&issues=AWgtKi6kXONnvcWD-k8f&open=AWgtKi6kXONnvcWD-k8f)

[ISSUE_KEY]: AWgtKi6kXONnvcWD-k8f